pkg/client/v1/writequery.go

```diff
@@ -70,14 +70,51 @@ func (c *Client) Create(ctx context.Context, schema string, opts ...CreateOption
 }
 
 // Write initiates a write query, returning the txn hash.
-func (c *Client) Write(ctx context.Context, query string) (string, error) {
```

Comment:
The rest is mostly creating options in the client and underlying (internal) client.